easybuild/easyconfigs/c/Coot/Coot-0.8.1-binary-Linux-x86_64-rhel-6-python-gtk2.eb

```diff
@@ -18,6 +18,9 @@ toolchain = {'name': 'dummy', 'version': ''}
 source_urls = ['http://www2.mrc-lmb.cam.ac.uk/Personal/pemsley/coot/binaries/release']
 sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tar.gz']
 
+postinstallcmds = ['chmod -R a=u %(installdir)s',
+                   'chmod -R g-w,o-w %(installdir)s']
```

Comment:
can you reformat this slighty?

```python
postinstallcmds = [
    'chmod -R a=u %(installdir)s',
    'chmod -R g-w,o-w %(installdir)s',
]
```

and maybe also add a comment (in the easyconfig) what you're fixing exactly?

Also, do you need to use two separate commands?